mobile/src/main/java/org/openhab/habdroid/core/connection/ConnectionFactory.kt

```diff
@@ -446,8 +524,8 @@ class ConnectionFactory internal constructor(
             instance.triggerConnectionUpdateIfNeededAndPending()
             val sub = instance.stateChannel.openSubscription()
             do {
-                val (available, reason, cloudInitialized, _) = sub.receive()
```

Comment:
Need to add `PrefKeys.PRIMARY_SERVER_ID` to `UPDATE_TRIGGERING_KEYS`? (I cannot comment on the correct line since it isn't changed in this commit).